mhs-reference-implementation/mhs/config/interactions.py

```diff
@@ -0,0 +1,24 @@
+import json
+
+
+class InteractionsFile:
```

Comment:
I am wondering if naming this class as `InteractionsConfig` would be better here, since this is specifically used to load the config file and get the resulting config details based on the interaction name?

Reply:
My goal was to reference the fact that this is a file-based implementation, as I think we'll ultimately replace it with something that goes and looks up interaction details in the SDS. Would something like `InteractionsConfigFile` be a bit clearer?

Reply:
Yeah I think that would be a little more clearer to make it obvious that it is only concerned with the Config of the interactions and not the actual interactions